src/gordon_gcp/plugins/janitor/authority.py

```diff
@@ -139,9 +143,13 @@ def __init__(self, config, crm_client, gce_client, rrset_channel=None,
     async def _get_projects(self):
         projects = await self._get_active_project_ids()
         project_blacklist = set(self.config.get('project_blacklist', []))
-        # TODO: emit a metric for all projects and a metric for
-        # project - blacklist.
-        return sorted(projects - project_blacklist)
+        sorted_projects = sorted(projects - project_blacklist)
+        context = {
+                'plugin': 'gceauthority'
+            }
+        await self.metrics.set(
+            'janitor-projects', len(sorted_projects), context=context)
```

Comment:
What do you think of making use of metrics config ([example](https://github.com/spotify/gordon/blob/develop/gordon/metrics/ffwd.py#L29)), and setting the key to be `gordon-janitor`, so then we don't have to prefix the metric names with `janitor-` everywhere. 

Also, since this is GCP-specific, if/when we have another provider(s), we may want to make it more obvious where it makes sense, e.g. `gcp-projects`, `gcp-plugin-runtime`, etc.